retriever/lib/engine_tools.py

```diff
@@ -162,6 +164,21 @@ def xml2csv(input_file, outputfile=None, header_values=None, row_tag="row"):
     return outputfile
 
 
+def xlsxcsv(book):
```

Comment:
The name should be `xslx2csv` to be consistent with other functions. Also, worth adding some comments to the code here.